functions/index.js

```diff
@@ -292,7 +318,7 @@ function toDataByLocation(data) {
   const cityIndex = headers.findIndex( e => e === 'city' );
   const data_by_location = {};
 
-  const published_entries = data.values.slice(1).filter((entry) => entry[approvedIndex] === "x");
+  const published_entries = data.values.slice(1);
```

Comment:
If we remove this filter here, we need to change 279 to take approved_rows...which is probably more correct anyways.

Reply:
ping! I think this still needs to be resolved.